spec/features/user_signs_up_spec.rb

```diff
@@ -18,6 +18,10 @@
     fill_in 'Email', with: 'john@example.com'
     fill_in 'Password', with: 'password'
     fill_in 'Password confirmation', with: 'password'
+    fill_in 'Username', with: 'john'
+    fill_in 'First name', with: 'John'
+    fill_in 'Last name', with: 'Smith'
+
 
```

Comment:
Lets remove one of the extra empty lines here.  Either 24 or 25

Reply:
We made this change.